api/v1alpha1/imagerepository_types.go

```diff
@@ -62,6 +62,11 @@ type ImageRepositoryStatus struct {
 	// LastScanResult contains the number of fetched tags.
 	// +optional
 	LastScanResult ScanResult `json:"lastScanResult,omitempty"`
+
+	// LastHandledScanRequest reocrds the value of the annotation
```

Comment:
```suggestion
	// LastHandledScanRequest records the value of the annotation
```

This will need a rebuild of the CRD (because the comment in the CRD is derived from this).

Reply:
But, I wonder if this shouldn't be named `LastHandledReconcileAt` https://github.com/fluxcd/kustomize-controller/blob/master/controllers/kustomization_controller.go#L264

Reply:
> I wonder if this shouldn't be named `LastHandledReconcileAt`

Mm, yeah probably. I went with my initial instinct, to name it something meaningful in its context, given the [discussion over the annotation](https://github.com/fluxcd/toolkit/discussions/363). But: it is not especially visible, so better the name be consistent.

Reply:
I read the discussion about naming, and yes, `reconcileAt` does feel a bit weird (it's like scheduling, but not), but I think consistency should win out (even if that means changing _other things_)